src/main.c

```diff
@@ -2425,6 +2428,28 @@ void handle_movement(double dt) {
         if (glfwGetKey(g->window, GLFW_KEY_RIGHT)) s->rx += m;
         if (glfwGetKey(g->window, GLFW_KEY_UP)) s->ry += m;
         if (glfwGetKey(g->window, GLFW_KEY_DOWN)) s->ry -= m;
+   
+       if (run_key_state == GLFW_PRESS){
+	
+	if(*is_running_ptr == false && *allow_next_run_key_press_ptr == true)
+	{
+		*movement_speed_ptr = 10;
+		*is_running_ptr = true;
+	}
+
+	else if(*is_running_ptr == true && *allow_next_run_key_press_ptr == true)
+	{
+		 *movement_speed_ptr = 5;
+		 *is_running_ptr = false;
+	}
+	*allow_next_run_key_press_ptr = false;
+       }
+	
+       else if (run_key_state == GLFW_RELEASE)
+	{
+		*allow_next_run_key_press_ptr = true;
+	}
+
```

Comment:
curly braces in front of if and else if statements should be formatted uniformly. Make a format of curly braces similar to line 2432.

Reply:
I'll consider this later. However, since it is not "crucial" for the functionality. For now I will not be implementing this change.